README.md

```diff
@@ -11,7 +11,7 @@
 This repo is for Spring Boot Starters of Azure services. It helps Spring Boot developers to adopt Azure services.
 
 ### Support Spring Boot
-This repository supports both Spring Boot 1.5.x and 2.0.x. Please read [this document](https://github.com/Microsoft/azure-spring-boot/wiki/Spring-Boot-dependency-version-management) for branch mapping.

```

Comment:
Can we also add badges for each of 2.0 starters? That will be more clear than just text here.

Reply:
Make sense, I search for a while for that badge in the morning but cannot tell how to play right now. I will try it later.

Reply:
Found this from shields.io:
![image](https://user-images.githubusercontent.com/2300784/50814803-db114200-1355-11e9-9ad2-073dbcf13e35.png)

Reply:
OK. I will try it later.